frb_codegen/src/error.rs

```diff
@@ -19,6 +19,10 @@ pub enum Error {
     MissingExe(String),
     #[error("{0}")]
     StringError(String),
+    #[error("please add {0} to your {1}. (version {2})")]
+    MissingDep(String, String, String),
+    #[error("please update version of {0} in your {1}. (version {2})")]
+    InvalidDep(String, String, String),
```

Comment:
uber-nit: shall we make it named fields

Reply:
Makes sense 👌

Reply:
done in f8f968b4c3eb1ff7d0cca2895ba0cfa071a5aae8